src/Rings/msolve/interface.jl

```diff
@@ -2,6 +2,77 @@
 # an int array lengths storing the lengths of each generator
 # an int array cfs storing the coefficients of each generator
 # an int array exps storing the exponent vectors of each generator
+function convert_oscar_ideal_to_array(
+        I::MPolyIdeal)
+
+    R = base_ring(I)
+    nr_vars     = nvars(R)
+    nr_gens     = ngens(I)
+    field_char  = characteristic(R)
+    lens        = Int32[length(I[i]) for i in 1:nr_gens]
+    nterms      = sum(lens)
+
+    # get coefficients
+    if field_char == 0
+        cfs = BigInt[]
+    else
+        cfs = Int32[]
+    end
+    if field_char == 0
+        for i in 1:nr_gens
+            for cf in coefficients(I[i])
+                push!(cfs, BigInt(numerator(cf)))
+                push!(cfs, BigInt(denominator(cf)))
+            end
+        end
+    else
+        for i in 1:nr_gens
+            for cf in coefficients(I[i])
+                push!(cfs, Int32(cf.data))
+            end
+        end
+    end
+
+    # get exponent vectors
+    exps  = Int32[]
+    for i in 1:nr_gens
+        for ev in exponent_vectors(I[i])
+            append!(exps, convert(Vector{Int32}, ev))
+        end
+    end
+
+    return lens, cfs, exps
+end
+
+function convert_ff_gb_array_to_oscar_array(
+        bld::Int32,
+        blen::Array{Int32,1},
```

Comment:
```suggestion
        blen::Vector{Int32},
```
We usually write `Vector` these days (and `Matrix{T}` for `Array{T,2}`), as this is what Julia does sine 1.6. However, this is not a blocker for this PR, I just wanted to point it out, feel free to change it later / never